.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
this file contains configuration for your local development environment. not everyone will want the same settings, so I'd gitignore this file. because the file is not project-specific, but rather specific to the setup of the developer, we tend to not put these into the projects `.gitignore`, but a global one on your local machine. see how to set one up here: https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#create-a-global-gitignore

as the file is empty, you can as well just delete it :)

Reply:
Thanks for the clarification @masone 👍